src/components/pages/FavoriteFoods.tsx

```diff
@@ -1,79 +1,106 @@
+import { DragDropContext, Draggable, Droppable } from '@hello-pangea/dnd'
 import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
 import Container from '@mui/material/Container'
-import Stack from '@mui/material/Stack'
 import Typography from '@mui/material/Typography'
-import { useContext } from 'react'
-import { DragDropContext, Draggable } from 'react-beautiful-dnd'
-import { Link } from 'react-router-dom'
+import { useContext, useState } from 'react'
+import { Link, useParams } from 'react-router-dom'
+import { FavoriteFood } from '../../client/flavorite'
 import useFavorites from '../../hooks/useFavorites'
 import useUpdateFavorites from '../../hooks/useUpdateFavorites'
-import { StrictModeDroppable } from '../helpers/strictModeDroppable'
 import AddFavorite from '../partials/AddFavorite'
 import Spinner from '../partials/Spinner'
 import { UserContext } from '../partials/UserContext'
 
 export default function FavoriteFoods() {
   const { currentUser } = useContext(UserContext)
-  const username = currentUser!.username
+  const { username } = useParams()
+  const profileUsername = username!
   const {
     favorites,
     loading: loadingFavorites,
     error: errorFavorites,
-  } = useFavorites({ username: username })
+  } = useFavorites({ username: profileUsername })
   const { error: errorUpdateFavorites, mutate: updateFavorites } = useUpdateFavorites()
+  const [favsList, setFavsList] = useState<FavoriteFood[]>(favorites)
 
-  const favoritesList = favorites.map(({ id, name }, idx) => {
-    return (
-      <Draggable key={id} draggableId={`${id}`} index={idx}>
-        {(provided) => (
-          <Stack
-            spacing={2}
-            ref={provided.innerRef}
-            {...provided.draggableProps}
-            {...provided.dragHandleProps}
-          >
-            <Typography>
-              {id}.<Link to={`/${username}/favorites/${name}`}>{name}</Link>
-            </Typography>
-          </Stack>
-        )}
-      </Draggable>
-    )
-  })
+  const handleSelection = () => {
+    // TODO styling view review button based on mouse hover or click
+    console.log('review pop up')
+  }
 
-  const handleUpdateFavorites = (result: any) => {
-    const items = Array.from(favorites)
+  const handleUpdateFavorites = async (result: any) => {
```

Comment:
specify result type instead of any.

Reply:
I just made this change in UnitTestFive (next PR) since I changed substantial changes in this component in the next PR for review.